autospec/specfiles.py

```diff
@@ -637,6 +637,29 @@ def write_mvnbin_install(self):
                     break
             src_num += 1
 
+    def write_maven_install(self):
+        self.write_license_files()
+
+        """Figure out the artifact details and path components."""
```

Comment:
More idiomatic is to use python comments beginning with `#` here, instead of docstrings. Same applies to the other comments you have added.